lib/brains/webhooks.py

```diff
@@ -19,6 +20,8 @@ class Processor(object):
 
     def __init__(self, webhook):
         self.webhook = webhook
+        self.transaction_ids = []
```

Comment:
I could be storing the transactions themselves instead of using ids and then doing a query. Mind you, I have to sort them by date.